dotnet/Directory.Packages.props

```diff
@@ -104,8 +104,8 @@
     <PackageVersion Include="MongoDB.Driver" Version="2.30.0" />
     <PackageVersion Include="Microsoft.Graph" Version="4.51.0" />
     <PackageVersion Include="Microsoft.Identity.Client.Extensions.Msal" Version="2.28.0" />
-    <PackageVersion Include="Microsoft.OpenApi" Version="1.6.22" />
-    <PackageVersion Include="Microsoft.OpenApi.Readers" Version="1.6.22" />
+    <PackageVersion Include="Microsoft.OpenApi" Version="2.0.0-preview2" />
```

Comment:
Since the 2.0 is a preview I strongly recomment we also move this package as a pre-release until the dependency is a release, unless extremely necessary to have another GA package using a non-GA dependency, this normally breaks the .Net Framework default installation.

Reply:
This PR is against the https://github.com/microsoft/semantic-kernel/tree/openapi-v2 feature branch which will be merged to main when the OpenAPI.NET V2 is GA/Released.